tasks/main.yml

```diff
@@ -16,26 +16,33 @@
 
 - name: apply kubectl service to create inception service
   command: |
-    {{bin_dir}}/kubectl apply -f {{ kube_config_dir }}/{{ item }}
+    {{kubectl_bin_dir}}/kubectl apply -f {{ kube_config_dir }}/{{ item }}
```

Comment:
Might be simpler to add `kubectl_bin_dir` to $PATH if it's not already, at least for the duration of this task?  Not sure if that's best Ansible practice though so I'll defer to others with better judgment here.

Reply:
Seems quite ugly: https://gist.github.com/carlessanagustin/af38e4585951dc32a320